integration/dbutils/migrator.spec.ts

```diff
@@ -12,6 +12,14 @@ var dbShape : Shapes.DBShape = {
   tables: [{
     tableName: 'users',
     indices: ['userName', 'email']
+  },
+  {
+    tableName: 'userData',
+    indices: []
+  },
+  {
+    tableName: 'activation',
```

Comment:
I thought about adding `activationCode` to `userData` table. But then, there needs to be an index on `userData` table for `activationCode` field to retrieve a user `id` from a given `activationCode`. At the time, it looked like a complex query and I just decided to add an `activation` table where you can lookup a user `id` by `activationCode`.

I can always put the `activationCode` in the `userData` table. Let me know if I should. It shouldn't be too much work.